test_scripts/Resumption/Handling_errors_from_HMI/011_unsubscribe_from_the_same_vehicledata_without_request_to_hmi_by_error_during_resumption.lua

```diff
@@ -0,0 +1,97 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Description:
+-- In case:
+-- 1. Subscriptions for data_1, data_2 are added by app1
+-- 2. Subscriptions for data_2 and data_3 are added by app2
+-- 3. Unexpected disconnect and reconnect are performed
+-- 4. App1 and app2 reregister with actual HashId
+-- 5. VehicleInfo.SubscribeVehicleData(data_2, data_3)requests for app2 is processed successful
+-- 6. VehicleInfo.SubscribeVehicleData(data_1, data_2) related to app1 is sent from SDL to HMI during resumption
+-- 7. HMI responds with error resultCode VehicleInfo.SubscribeVehicleData(data_1, data_2) request
+-- 8. HMI responds with success to remaining requests
+-- SDL does:
+-- 1. process unsuccess response from HMI
+-- 2. remove already restored data from app1
+-- 3. respond RegisterAppInterfaceResponse(success=true,result_code=RESUME_FAILED) to mobile application app1
+-- 4. restore all data for app2 and respond RegisterAppInterfaceResponse(success=true,result_code=SUCCESS)to mobile application app2
+---------------------------------------------------------------------------------------------------
```

Comment:
To check the difference with script 9

Reply:
Scripts 13 and 14 have been removed since the cases are the same as in 9 and 10
Scripts 11 and 12 have different scenario comparing to 9 and 10. In 9 and 10 error response is sent by HMI to shared data, but in 11 and 12 error response is sent to app1 data only

Reply:
Scripts 13 and 14 have been removed since the cases are the same as in 9 and 10
Scripts 11 and 12 have different scenario comparing to 9 and 10. In 9 and 10 error response is sent by HMI to shared data, but in 11 and 12 error response is sent to app1 data only